### DISCUSSION TITLE
Average outdoor temperature and LWT

### DISCUSSION THREAD

**Discussion by dlefol** [Arnold-n/P1P2MQTT#139] [Q&A] [answered]
Hello,

My heat pump is configured to work only based on the outside temperature and I have activated the averaging of sensor data over 24h so that the outside temperature should control the LWT based on the average outdoor temperature instead of instantaneous readings. I notice however that the value read with P1P2MQTT shows very quick variation of LWT based on the outside temperature (see red circles on the graphs below):

![LWT vs outside temperature](https://github.com/user-attachments/assets/dc2bc54b-03bc-44b5-8aff-2efa9fbfffef)

I am a bit astonished by these changes as I would expect the 24h averaging to smooth this out and the LWT not to change quickly like this. I am wondering if this is because I am not looking at the right sensor (maybe there is a different sensor after averaging ?). I use the sensors hc_sensors_temperature_outside and hc_lwt_lwt_setpoint which are provided by P1P2MQTT in home assistant.

This is not really an issue but I would like to understand better what happens so any input is welcome :)

Thanks a lot

**Comment by Staacks** [comment, marked as answer]
You probably have the power saving option of the outdoor unit enabled (field E-08). The heatpump won't do the averaging as long as this is on. (Never found any mention of this behavior in the manual - welcome to the mysterious world of Daikin's software...)

**Reply by Arnold-n** [reply]
True, though (some?) newer systems do not seem to have this strange limitation. Also, averaging does work even if the power saving option is activated if an (optional) external outside sensor is used instead of the outside unit's sensor - which has the additional benefit that it can be positioned such that the temperature measurement is not influenced by sun or by the cold/heat radiation of the outside unit's heat exchanger.

**Reply by dlefol** [reply]
Thanks. I have indeed the power saving mode activated and no other external sensor. I'll have to try it with the power saving option de-activated to see if it changes. Thanks a lot for the input. I had not think of this.

**Comment by forTT78** [comment]
Thx for the hint.
Textual description of [E-08]
![20241119_103827](https://github.com/user-attachments/assets/8ba8c5e9-863a-4b1c-bb00-0f4be25effdc)

![20241119_103834](https://github.com/user-attachments/assets/52b780ec-4d7f-4e75-94fc-2089ed0608d9)
